src/components/SelectInput/SelectInput.stories.tsx

```diff
@@ -210,12 +178,72 @@ export const MultiSelectCreatable = () => {
   );
 };
 
+export const AsyncSelect = () => {
+  const [value, setValue] = useState(null);
+  const filterOptions = (inputValue: string) => {
+    return options.filter((i) =>
+      i.label.toLowerCase().includes(inputValue.toLowerCase())
+    );
+  };
+  const loadOptions = (inputValue: string) => {
+    return new Promise<Option[]>((resolve) => {
+      setTimeout(() => {
+        resolve(filterOptions(inputValue));
+      }, 1000);
+    });
+  };
+
+  return (
+    <div style={{ height: '200px' }}>
+      <SelectInput
+        id="asyncSelect"
+        label="Label"
+        value={value}
+        // @ts-ignore - TS is not recognizing the value as a valid option
+        onChange={(event) => setValue(event.target.value)}
+        options={loadOptions}
+        defaultOptions
+        cacheOptions
+        isAsync
+      />
+    </div>
+  );
+};
+
+export const AsyncCreatableSelect = () => {
+  const [value, setValue] = useState(null);
+  const filterOptions = (inputValue: string) => {
+    return options.filter((i) =>
+      i.label.toLowerCase().includes(inputValue.toLowerCase())
+    );
+  };
+  const loadOptions = (inputValue: string) => {
+    return new Promise<Option[]>((resolve) => {
+      setTimeout(() => {
+        resolve(filterOptions(inputValue));
+      }, 1000);
+    });
+  };
+
+  return (
+    <div style={{ height: '200px' }}>
+      <SelectInput
+        id="asyncCreateSelect"
+        label="Label"
+        value={value}
+        // @ts-ignore - TS is not recognizing the value as a valid option
+        onChange={(event) => setValue(event.target.value)}
+        options={loadOptions}
+        isCreatable
+        defaultOptions
+        cacheOptions
+        isAsync
+      />
+    </div>
+  );
+};
```

Comment:
Can you update these stories so that the defaultOptions is different than the async options (maybe a list of 10 flavors) so it's more obvious that more has been loaded?